--- argv_emulation.py	2018-02-01 16:29:25.000000000 -0800
+++ argv_emulation.py.new	2018-02-01 16:30:10.000000000 -0800
@@ -254,7 +254,7 @@
 
         sts = carbon.ReceiveNextEvent(
             1, ctypes.byref(eventType),
-            start + timeout[0] - now, TRUE, ctypes.byref(event))
+            start + timeout[0] - now, FALSE, ctypes.byref(event))
 
         if sts == eventLoopTimedOutErr:
             break
